tests/mtx/dlms/get.day.demand.test.ts

```diff
@@ -34,8 +33,8 @@ const examples = [
                 month: 2,
                 date: 3
             },
-            energyType: A_PLUS_ENERGY_TYPE,
-            energies: [40301230, undefined, undefined, 2145623]
+            energyType: 1,
```

Comment:
why remove constant?

Reply:
constants restored as file constants/energyTypes.ts, added examples in TSdoc

Reply:
how about use this file here as well?

Reply:
added